Client/Layout/MainLayout.razor

```diff
@@ -2,15 +2,17 @@
 
 <RadzenComponents />
 <div class="page">
-    <div class="sidebar">
-        <NavMenu />
-    </div>
-
     <main>
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <NavBar />
         </div>
-
+        <div class="search-container">
```

Comment:
Will this search be present on all pages or just on the home page?

Reply:
all pages